src/diagnostics/radiation_diagnostics.jl

```diff
@@ -96,12 +96,12 @@ function compute_rld!(
     FT = eltype(state)
     if isnothing(out)
         return RRTMGPI.array2field(
-            FT.(cache.radiation_model.face_lw_flux_dn),
+            FT.(cache.atmos.radiation.radiation_model.face_lw_flux_dn),
```

Comment:
Is `cache.atmos.radiation` the same as `cache.radiation`? (I was wondering why this line is different from e.g. line23)

On a side note, we didn't test all the diagnostics in ci, so it would be hard to catch things like this.